packages/tools/fetch-tool/src/fluidFetchSnapshot.ts

```diff
@@ -243,19 +244,21 @@ async function saveSnapshot(name: string, fetchedData: IFetchedData[], saveDir:
 
         if (!isFetchedTree(item)) {
             fs.writeFileSync(`${outDir}/${item.filename}`, data);
-            const decoded = bufferToString(buffer,"utf8");
+            let decoded = bufferToString(buffer,"utf8");
             try {
-                const object = JSON.parse(decoded);
-                fs.writeFileSync(`${outDir}/decoded/${item.filename}.json`, JSON.stringify(object, undefined, 2));
+                if (!paramActualFormatting) {
+                    decoded = JSON.stringify(JSON.parse(decoded), undefined, 2);
+                }
             } catch (e) {
-                fs.writeFileSync(`${outDir}/decoded/${item.filename}.txt`, decoded);
             }
+            fs.writeFileSync(
+                `${outDir}/decoded/${item.filename}.json`, decoded);
```

Comment:
So here we were writing in .txt if we were having issues with json. Don't we need that anymore?

Reply:
We will write the file no matter what, and it will have .json extension, no matter if it's actually json or not.
I feel like better design would be to not differentiate and not assume it's json (i.e. extension to be .txt or something). We can change if if we want to, it's just was .json for all practical purposes, so I left it as is